project_1.html

```diff
@@ -0,0 +1,421 @@
+				The voltage is noted in the LED’s datasheet.
+			</p>
+
+			<h3 class="mb-3" id="pj-heading-concepts-slide-switch">Slide Switch</h3>
+
+			<p class="mb-4">
+				<img class="img-fluid my-3 me-5 float-start" style="max-width: 400px;"
+					src="./assets/images/project_1/ops_switch_slide_annotated.png">
+				You will use a <strong>slide switch</strong> to control the current flow in the circuit. The sliding contact completes a
+				connection with the middle terminal (pin 2) and one of the two outer terminals (pins 1 and 3).
+				<br><br>
+				The slider is shifted in the direction of the outer terminal actively connected to the middle terminal.
+			</p>
+
+			<h3 class="mb-3" id="pj-heading-concepts-soldering">Soldering</h3>
+			<p class="mb-5">
+				Woah woah woah! Soldering is a bit more complicated than a two paragraph discussion. 
+				Head over to the workshop material <span class="text-blue">here [Webpage Link]</span> to learn more.
+			</p>			
+
+			<h2 class="mb-3 fw-bolder" id="pj-heading-requirements">Requirements</h2>
+			<ul class="mb-5">
+				<li>The circuit must work as implied by the schematic: A switch will toggle the LED ON and OFF.</li>
+				<li>The circuit must be soldered to the perfboard.</li>
+				<li>The LED must be brightly lit when toggled ON. A dim LED indicates poor soldering or an incorrect resistor value.</li>
+				<li>The LED must be completely dark when toggled OFF.</li>
+			</ul>
+
+			<h2 class="mb-3 fw-bold" id="pj-heading-parts">Parts</h2>
+
+			<!-- ======= Part Table ======= -->
+			<table class="table pj-part-table mb-5">
+				<thead>
+					<tr>
+						<th scope="col">Part Name</th>
+						<th scope="col">Qty</th>
+					</tr>
+				</thead>
+				<tbody>
+					<tr>
+						<td>
+							Jumper Wire
+						</td>
+						<td>X?</td>
+					</tr>
+					<tr>
+						<td>
+							Breadboard
+						</td>
+						<td>1</td>
+					</tr>
+					<tr>
+						<td>
+								Battery, 9V
+						</td>
+						<td>1</td>
+					</tr>
+					<tr>
+						<td>
+							<a class="pj-part"
+								data-bs-content="<img src='./assets/images/parts/ops_part_batterySnap.jpg' height='150px' width='auto'>"
+								data-bs-trigger="hover focus" data-bs-toggle="popover" tabindex="0">
+								9V Snap Connector
+							</a>
+						</td>
+						<td>1</td>
+					</tr>
+					<tr>
+						<td>
+							<a class="pj-part"
+								data-bs-content="<img src='./assets/images/parts/ops_part_2pinHeader.jpg' height='150px' width='auto'>"
+								data-bs-trigger="hover focus" data-bs-toggle="popover" tabindex="0">
+								Header, 2.54mm, Female, 1x2
+							</a>
+						</td>
+						<td>2</td>
+					</tr>
+					<tr>
+						<td>
+							<a class="pj-part"
+								data-bs-content="<img src='./assets/images/parts/ops_part_led.png' height='150px' width='auto'>"
+								data-bs-trigger="hover focus" data-bs-toggle="popover" tabindex="0">
+								LED, 3.2V
```

Comment:
Can we include the datasheet link here? I'm thinking of a blue link indicating that it can be clicked on. It would still have the dashed line to indicate it is hoverable.

Reply:
done